src/site/content/en/progressive-web-apps/offline-fallback-page/index.md

```diff
@@ -0,0 +1,297 @@
+  // event.respondWith(), the request will be handled by the browser as if there
+  // were no service worker involvement.
+});
+```
+
+### The offline fallback page
+
+The `offline.html` file is where you can get creative and adopt it to your needs and add your
+branding. The example below shows the bare minimum of what is possible.
+
+{% Aside 'gotchas' %} Note that you need to cache all resources required by your offline page. One
+way to deal with this is to inline everything, so the offline page is self-contained. This is what I
+do in the example below. {% endAside %}
+
+```html
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+
+    <title>You are offline</title>
+
+    <!-- inline the webpage's stylesheet -->
+    <style>
+      body {
+        font-family: helvetica, arial, sans-serif;
+        margin: 2em;
+      }
+
+      h1 {
+        font-style: italic;
+        color: #373fff;
+      }
+
+      p {
+        margin-block: 1rem;
+      }
+
+      button {
+        display: block;
+      }
+    </style>
+  </head>
+  <body>
+    <h1>You are offline</h1>
+
+    <p>Click the button below to try reloading.</p>
+    <button type="button">⤾ Reload</button>
+
+    <!-- inline the webpage's javascript file -->
+    <script>
+      document.querySelector("button").addEventListener("click", () => {
+        window.location.reload();
+      });
+    </script>
+  </body>
+</html>
+```
+
+## Demo
+
+You can see the offline fallback page in action in the
+[demo](https://offline-fallback-demo.glitch.me/index.html) that is also embedded below. If you are
+interested, you can explore the [source code](https://glitch.com/edit/#!/offline-fallback-demo) on
+Glitch.
+
+<div class="glitch-embed-wrap" style="height: 420px; width: 100%;">
+  <iframe
+    src="https://glitch.com/embed/#!/embed/offline-fallback-demo?path=offline.html&previewSize=100"
+    title="offline-fallback-demo on Glitch"
+    allow="geolocation; microphone; camera; midi; vr; encrypted-media"
+    style="height: 100%; width: 100%; border: 0;">
+  </iframe>
+</div>
+
+### Side note on making your app installable
+
+Now that your site has an offline fallback page, you might wonder about next steps. In order to make
+your app installable, you need to add a [Web App Manifest](/add-manifest/) and optionally come up
+with an [install strategy](/define-install-strategy/).
+
+### Side note on serving an offline fallback page with Workbox.js
+
+You may have heard of [Workbox.js](https://developers.google.com/web/tools/workbox) in the past.
+Workbox.js is a set of JavaScript libraries for adding offline support to web apps. If you prefer to
+write less service worker code yourself, you can use the Workbox.js recipe for an
+[offline page only](https://developers.google.com/web/tools/workbox/guides/advanced-recipes#offline_page_only).
```

Comment:
This ends abruptly. Do you plan to eventually add links between articles in this series? 
![image](https://user-images.githubusercontent.com/8204171/93778018-48244400-fbda-11ea-93af-05c37d4b107c.png)

Reply:
These "series" articles end kind of abruptly indeed, and right now the navigation isn't really straightforward. There is an "appears in: $seriesName" at the beginning…

<img width="692" alt="Screen Shot 2020-09-21 at 16 47 29" src="https://user-images.githubusercontent.com/145676/93781960-43877c00-fc2a-11ea-91df-7f11e3927cc9.png">

…but I think there should be a link to the next article in the series in the footer. Feedback for the web.dev core team.

Reply:
Let me add a pointer manually for now.